### PR TITLE
PR for new audio settings

### PR DESCRIPTION
**Pull request by tphan5901**
Add new settings for users to add/save and play audio files

### CODE REVIEW

File: index-mobile.html [hunk 1]
```diff
@@ -88,6 +92,7 @@
     <script src="ol3dc/js/interpolator/interpolator.js"></script>
     <script src="ol3dc/js/entry/entry-manager.js"></script>
     <script src="ol3dc/js/effect/blur.js"></script>
+    <script src="ol3dc/js/effect/backgroundblur.js"></script>
```
Comment (Wei-1) [inline index-mobile.html:95]: let's remove unrelated changes

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <canvas class="backgroundeffect" id="backgroundeffect"></canvas>
     </div>
     <div class="bgimg" id="bgimg"></div>
-    <div class="w3-sidebar w3-animate-left" style="width:270px;background-color:#0006;display:none" id="thesidebar">
+    <div class="w3-sidebar w3-animate-left" style="width:400px;background-color:#0006;display:none" id="thesidebar">
```
Comment (Wei-1) [inline index.html:20]: For synchronized style, let's keep the `width:270px`.
As long as the style is updated in the `.core` repository, we should be good with 270.

File: index.html [hunk 3]
```diff
@@ -88,18 +93,21 @@
     <script src="ol3dc/js/interpolator/interpolator.js"></script>
     <script src="ol3dc/js/entry/entry-manager.js"></script>
     <script src="ol3dc/js/effect/blur.js"></script>
+    <script src="ol3dc/js/effect/backgroundblur.js"></script>
```
Comment (Wei-1) [inline index.html:96]: remove unrelated change.

File: index.html [hunk 4]
```diff
@@ -88,18 +93,21 @@
...
     <script src="ol3dc/js/effect/effect-manager.js"></script>
     <script src="ol3dc/js/config/config-manager.js"></script>
     <script src="ol3dc/js/gui/multi-language.js"></script>
-    <script src="ol3dc/js/gui/gui-layout.js"></script>
     <script src="ol3dc/js/vrm/vrm-manager.js"></script>
     <script src="ol3dc/js/control/health-manager.js"></script>
     <script src="ol3dc/js/control/control-manager.js"></script>
     <script src="ol3dc/js/control/hotkey.js"></script>
+    <script src="ol3dc/js/gui/gui-layout.js"></script>
+    <script src="ol3dc/js/clientStorage/index.js"></script>
```
Comment (Wei-1) [inline index.html:109]: Let's move this import higher up, since this is independent from others.

**Review by Wei-1 — COMMENTED** [review]
Updated a few changes corresponding to the change in: https://github.com/OpenLive3D/OpenLive3D.core/pull/9
This is a note that the 2 PRs are connected.

File: index.html (hunk 2 above)
Comment (tphan5901) [reply index.html:20]: Generally no problem with suggested changes, one critique is sometimes the sidebar is too small. Whenever u go into sub settings in general, it will get cut off from the sidebar view.

File: index.html (hunk 4 above)
Comment (tphan5901) [reply index.html:109]: I will do that :)

File: index.html (hunk 2 above)
Comment (Wei-1) [reply index.html:20]: Make sense, let's go with your suggested 330.
